turbinia/evidence.py

```diff
@@ -282,12 +304,31 @@ def __init__(self, project=None, zone=None, disk_name=None, *args, **kwargs):
     super(GoogleCloudDisk, self).__init__(*args, **kwargs)
     self.cloud_only = True
 
+    self._attached_path = None
+
   def _preprocess(self):
-    self.local_path = google_cloud.PreprocessAttachDisk(self.disk_name)
+    self._attached_path = google_cloud.PreprocessAttachDisk(self.disk_name)
+    self.loopdevice_path = mount_local.PreprocessLosetup(self._attached_path)
```

Comment:
Could we add a TODO(aarontp): Remove PreprocessLosetup() once https://github.com/google/turbinia/issues/78 is fixed.

Reply:
TODOs have been sprinkled